### PR TITLE
Add new tutorials to docs

### PR DESCRIPTION
**Pull request by tr0njavolta**

### CODE REVIEW

File: website/docs/configuration/expressions.html.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ description: |-
 earlier, see
 [0.11 Configuration Language: Interpolation Syntax](../configuration-0-11/interpolation.html).
 
+> **Hands-on:** Try the [Create Dynamic Expressions](https://learn.hashicorp.com/tutorials/terraform/expressions?in=terraform/configuration-language&utm_source=WEBSITE&utm_medium=WEB_IO&utm_offer=ARTICLE_PAGE&utm_content=DOCS) tutorial on HashiCorp Learn.
```
Comment (apparentlymart) [inline website/docs/configuration/expressions.html.md:16]: If you use `->` as a prefix here, rather than `>`, then the custom Markdown extension we have in the website renderer will present it as a blue "information" callout box rather than as a blockquote, which I think would make it a little more prominent on the page, assuming that prominence is desirable. :grinning: 

(With that said, in this particular case I suppose there is _already_ a blue callout box right above this, so maybe that would just look _weird_... :confused: )

Comment (tr0njavolta) [reply website/docs/configuration/expressions.html.md:16]: Hmm.. I was going with the formatting from https://www.terraform.io/docs/configuration/variables.html but it does indeed look weird

Comment (nfagerlund) [reply website/docs/configuration/expressions.html.md:16]: Yeah, we went back and forth on this when I was originally doing these "hands-on" callouts, and Judith and I settled on (mis)using the blockquote syntax to set it apart in a less obtrusive way. They tend to go near-ish to a blue box, so we didn't want to create too much blue box chaos. 

tl;dr plain `>` is correct in this case.

**Review by nfagerlund — APPROVED** [review]
This is going to collide with https://github.com/hashicorp/terraform/pull/26902, so whichever one of us gets merged second will have to transfer these to their new homes. (I'm fine with that being me, though.)

**Comment by nfagerlund** [comment]
Superseded by https://github.com/hashicorp/terraform/pull/27066, which puts these links on the relevant broken-out pages.
